### DISCUSSION TITLE
send inscription (ordinals) transaction

### DISCUSSION THREAD

**Discussion by amirmovafagh** [1200wd/bitcoinlib#367] [Q&A] [answered]
Hi, is it possible to create an ordinal transaction with bitcoinlib and broadcast it on the network?

**Comment by mccwdev** [comment, marked as answer]
This is not possible at this moment as it requires taproot, which is still in development for this library.

And I do not have any technical knowledge about BRC-20, but you can create custom transactions and add data to transactions, so once taproot becomes available I assume you can create BRC-20 transactions as well.
